packages/react-router-dom/modules/Link.js

```diff
@@ -52,9 +57,24 @@ class Link extends React.Component {
     }
   }
 
-  render() {
-    const { replace, to, ...props } = this.props // eslint-disable-line no-unused-vars
+  absolute() {
+    const { to } = this.props
+    const { match } = this.context.router
+    const base = (match && match.url) ? match.url : ''
+    return resolveLocation(to, base)
+  }
+
+  componentWillMount() {
+    this.unlisten = this.context.router.listen(() => { this.forceUpdate() })
```

Comment:
I honestly can't even consider redux users when working on the router. They do the craziest things in their apps and show up with all sorts of requirements that distract me from making the core of the router good. If the router is good React code, then the redux people will be fine. If it's not, then they'll get all creative and their constraints will make our codebase worse.